### ISSUE TITLE
accessibility guides, tips, examples, statement

### ISSUE THREAD

**Issue by evanwill** [CollectionBuilder/cb-docs#28]
things to add to docs:
- "accessibility tab" -- lays "out what aspects of CB meet WCAG 2.1AA guidelines, which aspects of CB partially meet 2.1AA, and which aspects fail to meet 2.1AA... This could also be a space where you point out any accessibility features that CB has. You could also even include a demo about, from the perspective of a CB maker, how they could make choices to make more accessible projects." (Elliot)
- accessibility tips for creators in each relevant section (e.g. how to write about page using headers correctly, how to make good color choices during customization)

examples: 
- [manifold statement](https://manifoldscholar.github.io/manifold-docusaurus/docs/accessibility/)
- [pressbooks](https://pressbooks.com/accessibility/)
- [fulcrum](https://www.fulcrum.org/accessibility/)

resources: 
- [Manifold Key Concepts for Accessibility](https://uw.manifoldapp.org/projects/manifoldaccess)
- [pressbooks, Make Your Book Accessible and Inclusive](https://guide.pressbooks.com/chapter/make-your-book-accessible-and-inclusive/)
- [no mouse challenge](https://nomouse.org/)
- [DLF accessibility working group](https://wiki.diglib.org/Digital_Accessibility_Group), [Accessibility Auditing Shortlist](https://wiki.diglib.org/Accessibility_Auditing_Shortlist)

**Comment by evanwill** [comment]
section started in PR #30
